code/dictation.py

```diff
@@ -143,41 +171,98 @@ def __init__(self):
         self.reset()
 
     def reset(self):
+        self.reset_except_forced()
+        self.force_no_space = False
+        self.force_cap = False
+        self.force_no_cap = False
+
+    def reset_except_forced(self):
         self.before = ""
-        self.state = "sentence start"
+        self.state = None
```

Comment:
Note that this is a behavior change in dictation mode, which now no longer starts with a capitalized letter, consistent with modeless "speak" command. We could easily avoid this if desired.

Reply:
Could we make this a configuration option? I don't see need to change the default behavior, but that could make it easy for people to pick what they like.

Reply:
That would certainly work for me. I imagine we'd would want to preserve the "speak" behavior instead of aligning it. That gives us two options for implementation: either change what DictationFormat.reset does and special-case "speak" to set state to None, or update calls to DictationFormat.reset, perhaps by having them call user.dictation_format_reset() instead and putting this behavior in there. I'm weakly inclined towards the latter.

Reply:
Actually what we'd do in the former case is set formatter.state = None in apply_formatting() here, which I'm already doing. That's probably the simpler approach then.

Reply:
@phillco Do you prefer the current behavior?

Reply:
I do, although it's easy enough for me to change it in my fork.

Reply:
I do, although it's easy enough for me to change it in my fork.

Reply:
Reverted.